src/setup/cortx_deploy

```diff
@@ -28,15 +28,17 @@ function show_usage {
     echo -e "                   DEBUG=1, INFO=2, WARNING=3, ERROR=4, FATAL=5"
     echo -e " CORTX-CONFIG-URL: URL of CORTX Configuration"
     echo -e "                   (Default is yaml:///etc/cortx/cluster.conf)"
-    echo -e " ENABLE MOCK     : Enable Mocking CORTX Deployment (Default: False)"
+    echo -e " ENABLE FORCE     : Override deployment, forcefully execute config apply and bootstrap cmds."
```

Comment:
"Force deployment steps if it was executed earlier."

Reply:
Done